frontend/constants/width.ts

```diff
@@ -4,6 +4,7 @@ export const APP_HEIGHT = 796;
 export const TOP_BAR_HEIGHT = 44;
 
 export const MODAL_WIDTH = 412;
+export const MODAL_WIDTH_V1 = 512;
```

Comment:
same here, we could just update the value for existing MODAL_WIDTH not to get back here anymore